Minos/Minos.Site/Controllers/AdminController.cs

```diff
@@ -77,12 +77,30 @@ public IActionResult CadastrarTurma(Grau grau, Serie serie, Turno turno, string
             }
             else
             {
+                turma.Ativo = true;
                 _turmaRepository.Salvar(turma);
             }
 
             return View();
         }
 
+        [HttpPost]
+        public IActionResult DesativarTurma (int id)
+        {
+            if(id <= 0)
```

Comment:
id nunca será menor que 0.

Reply:
Então, já que não nunca vai ser menor, coloquei que se ele for menor que 0 ele entre no if, por que nunca teremos uma turma com id igual a 0 no banco se tiver é um erro.